build.gradle

```diff
@@ -123,11 +123,11 @@ buildScan {
 }
 
 ext {
-    compileSdkVersion = 29
+    compileSdkVersion = 30
     buildToolVersion = '29.0.2'
 
     minSdkVersion = 24
-    targetSdkVersion = 29
+    targetSdkVersion = 30
```

Comment:
Suggestion (💡): Since the buildToolVersion is not needed and actually not used anymore, we can remove that as well. Afterwards, with some reordering this might look like:
```
    minSdkVersion = 24
    compileSdkVersion = 30
    targetSdkVersion = 30
```

PS: This is the [same comment](https://github.com/Automattic/stories-android/pull/715/files#r708287446) copy-pasted from `stories-android`.

Reply:
- [ ] done, 🙏